Britannia/localization/english/BTWK_storyline_events_l_english.yml

```diff
@@ -1,36 +1,72 @@
+ BTWK_story_event.0118.a:0 "I should ask Agricola to carry forth my will."
+ BTWK_story_event.0118.tt_a:0 "Comes Agricola may gain a [weak_hook|E] on you."
+ BTWK_story_event.0118.b:0 "This isn't worth risking the Uther's ire."
+
+ BTWK_story_event.0117.t:0 "Tewdric's Request"
+ BTWK_story_event.0117.desc:0 "Before the High Council can make head, your liege Tewdric has come with a request. To avoid personally provoking the ire of High King Uther (and possibly even hostile kings of neighbouring realms), he asks you to petition Uther to arrange Prince Arthur's marriage to the Lady Norwenna. The young Prince had been banished by none other than Uther himself, his own father, following the death of the High King's only legitimate son and heir's death. Doing so will ensure that Arthur - a brilliant soldier and leader in his own right - may return to the shores of Britannia once more to aid in her struggle against the invading barbarians.\n\n To avoid risking any suspicion, you are to frame the act as the will of your fellow soldiers - easily done, given Arthur's popularity among them.    "
+ BTWK_story_event.0117.a:0 "As my lord requests."
+ BTWK_story_event.0117.b:0 "It isn't worth risking the ire of King Uther."
+
+ BTWK_story_event.0113.t:0 "Agricola's Suggestion"
+ BTWK_story_event.0113.desc:0 "DESC"
+ BTWK_story_event.0113.a:0 "#italic Arthur ap neb. #"
+ BTWK_story_event.0113.b:0 "As much as it pains me, he is right."
+
+ BTWK_story_event.0104.t:0 "Choosing Mordred's Fosterer"
+ BTWK_story_event.0104.desc:0 "The most pressing matter of discussion is finding Mordred a foster father. The warlord you choose to appoint to the role of Mordred's foremost guardian will in return benefit from a strong alliance with your kingdom. Only those who are unmarried, or rulers with unmarried heirs, are eligible however. In order to seal the pact, you plan to marry the chosen candidate to your daughter-in-law, the late Mordred's widow, and the younger Mordred's mother - Lady Norwenna.\n\n After much debate, the list of suitors has been whittled down to a small selection. Who will you choose?"
```

Comment:
"Only those who are unmarried, or rulers with unmarried heirs, are eligible however." -> Maybe make this into a more passing reference, e.g. "The need for a marriage alliance to seal the agreement restricts this to unmarried rulers or heirs, though hopefully this will not be too much of a hindrance." so it sounds more in context/character